plugins/multicamera/src/MultiCameraLog.cpp

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright (C) 2013-2021 Fondazione Istituto Italiano di Tecnologia RBCS & iCub Facility & ADVR
+ * Authors: see AUTHORS file.
+ * CopyPolicy: Released under the terms of the LGPLv2.1 or any later version, see LGPL.TXT or LGPL3.TXT
+ */
+
+#include "gazebo/MultiCameraLog.h"
+
+namespace GazeboYarpPlugins{
+    YARP_LOG_COMPONENT(GAZEBOMULTICAMERA, "gazebo-yarp-plugins.plugins.GazeboYarpMultiCamera MultiCamera.cc")
```

Comment:
The style of this seems different, also in the header, is this expected?

Reply:
cc @ste93

Reply:
sorry the comment was pending and it was not sent.

do you intend the include? that's because the cmake file in this class is different and doesn't include the gazebo directory by default.

Reply:
No, I mean the `MultiCamera.cc` part in the `YARP_LOG_COMPONENT` string.

Reply:
Ok I didn't noticed that, it was a refuse. I'll remove it.